atrope/dispatcher/glance.py

```diff
@@ -123,6 +169,15 @@ def dispatch(self, image_name, image, is_public, **kwargs):
         """
         LOG.info("Glance dispatching '%s'", image.identifier)
 
+        vos = kwargs.pop("vos")
+
+        if CONF.glance.sharing_model == 'community':
+            visibility = "community"
+        elif CONF.glance.sharing_model == 'shared' and vos:
+            visibility = "shared"
+        else:
```

Comment:
I see `is_public` being forced to `True` elsewhere in the code. I would just move this as another possible option for the `sharing_model`. So the options should be `community` (which is almost like `public`), `shared`, `public` and `private` for no sharing at all (although this would prevent that other VOs see the image).